Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -55,7 +61,7 @@ extension AnimeViewController: UISearchBarDelegate {
         self.searchTask?.cancel()
         
         let task = DispatchWorkItem { [weak self] in
-            DispatchQueue.global(qos: .userInteractive).async { [weak self] in
+            DispatchQueue.main.async {
```

Comment:
You shouldn't be calling this on the main thread, your networking manager should have the dispatch queue main in the networking manager so everything in that closure is on the main thread. You shouldn't be calling this in the view controller